src/jitlayers.cpp

```diff
@@ -244,6 +244,10 @@ void addOptimizationPasses(legacy::PassManagerBase *PM, int opt_level)
 #endif
 }
 
+void jl_add_optimization_passes(LLVMPassManagerRef PM, int opt_level) {
```

Comment:
ok. I guess there shouldn't be a prototype in `jitlayers.h` at all, cfr. many of the DLLEXPORT functions in eg. `codegen.cpp`.